src/sass/components/_footer.scss

```diff
@@ -0,0 +1,77 @@
+.footer {
+  background-color: $pageBg;
+  display: block;
+  flex-direction: column;
+  min-width: 320px;
+
+  &__header {
+    color: $brand-pink;
+    font-family: 'Titan One';
+    font-size: 18px;
+    font-weight: 400;
+    display: block;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    align-items: flex-start;
+    padding-top: 20px;
+    padding-left: 20px;
+    max-width: 80%;
+    &--color {
+      color: $brand-red;
+      font-size: 34px;
+      margin-top: 16px;
+      display: block;
+    }
+  }
+  &__list {
+    display: flex;
+    list-style: none;
```

Comment:
to już jest w stylach globalnych, usuń

Reply:
usunęłam, zobacz jak bardzo się wy****ło svg, no tatalnie to wygląda